assets/dev/js/editor/elements/models/column.js

```diff
@@ -0,0 +1,16 @@
+import Element from './element';
+
+export default class Column extends Element {
+	/**
+	 * Allow only widget, container and inner-section.
+	 */
+	isValidChild( childModel ) {
+		const childElType = childModel.get( 'elType' );
+
+		if ( 'section' === childElType && childModel.get( 'isInner' ) ) {
+			return true;
+		}
+
+		return 'widget' === childElType || 'container' === childElType;
```

Comment:
Just to be consistent with other places

```suggestion
		return [ 'widget', 'container' ].includes( childElType );
```